### PR TITLE
Private transcripts

### PR DESCRIPTION
**Pull request by stephencelis**
This is another attempt to maintaining a separate repo with private/production episodes.

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -0,0 +1,3 @@
+[submodule "Transcripts"]
+	path = Transcripts
+	url = git@github.com:mbrandonw/episode-transcripts.git
```
Comment (stephencelis) [inline .gitmodules:3]: We're doing the submodule way.

File: Makefile [hunk 2]
```diff
@@ -1,6 +1,14 @@
 start:
+	test -f .env || make local-config
+	test -d Packages || swift package edit PointFree
 	docker-compose up --build
 
+transcripts:
+	git submodule update --init --recursive
+	ln -fn Transcripts/*.swift Sources/pointfreeco/Transcripts
+	git update-index --assume-unchanged Sources/pointfreeco/Transcripts/
+	make xcodeproj
```
Comment (stephencelis) [inline Makefile]: Calling `make transcripts` puts our repo into private mode:

1. It checks out the private submodule
2. It copies over the Swift files in the root of that repo (and at the moment assumes clobbering **AllEpisodes.swift**)
3. It locally ignores changes in that directory (so we don't accidentally commit them to this public repo)
4. Regenerates the Xcode project

File: .gitignore [hunk 3]
```diff
@@ -1,3 +1,5 @@
+Sources/pointfreeco/Transcripts/*
```
Comment (stephencelis) [inline .gitignore]: This prevents new transcripts from being staged in this repo.

File: Sources/pointfreeco/Transcripts/AllEpisodes.swift [hunk 4]
```diff
@@ -0,0 +1,7 @@
+import PointFree
+
+func allEpisodes() -> [Episode] {
```
Comment (stephencelis) [inline Sources/pointfreeco/Transcripts/AllEpisodes.swift:3]: This is a stub that returns a stubbed episode for OSS. It gets overwritten by the private repo.

File: Makefile [hunk 5]
```diff
@@ -1,6 +1,18 @@
 start:
+	test -f .env || make local-config
```
Comment (stephencelis) [inline Makefile:2]: This stuff is required to run for Linux to work! Sorry for the noise!

**Review by mbrandonw — APPROVED** [review]
great!

**Pull request merged**